<ui version="4.0" >
 <class>frmUpload</class>
 <widget class="QDialog" name="frmUpload" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>242</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>540</width>
    <height>185</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>QNapi - dodawanie nowych napisów</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../res/resources.qrc" >
    <normaloff>:/icon/qnapi-48-icon.png</normaloff>:/icon/qnapi-48-icon.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QLabel" name="lbSelectDirectory" >
     <property name="text" >
      <string>Wskaż katalog, z którego dodać napisy:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QLineEdit" name="leSelectDirectory" />
   </item>
   <item row="1" column="3" >
    <widget class="QPushButton" name="pbSelectDirectory" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset resource="../res/resources.qrc" >
       <normaloff>:/ui/folder.png</normaloff>:/ui/folder.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="4" >
    <widget class="QPushButton" name="pbScan" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>&amp;Skanuj</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="5" >
    <widget class="QLabel" name="lbAction" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="5" >
    <widget class="QProgressBar" name="pbProgress" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="value" >
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="5" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="lbFoldersCount" >
       <property name="text" >
        <string>Katalogów: &lt;b>0&lt;/b></string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbMoviesCount" >
       <property name="text" >
        <string>Filmów: &lt;b>0&lt;/b></string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbSubtitlesCount" >
       <property name="text" >
        <string>Napisów: &lt;b>0&lt;/b></string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Anuluj</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbUpload" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Wyślij napisy do serwera</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="QCheckBox" name="cbFollowSymLinks" >
     <property name="text" >
      <string>Podążaj za dowiązaniami symbolicznymi</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>frmUpload</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>321</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel" >
     <x>267</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
